packages/crud/src/vaadin-crud-dialog.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2021 Vaadin Ltd.
+ * This program is available under Commercial Vaadin Developer License 4.0, available at https://vaadin.com/license/cvdl-4.0.
+ */
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { Dialog, DialogOverlay } from '@vaadin/dialog/src/vaadin-dialog.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+registerStyles(
+  'vaadin-crud-dialog-overlay',
+  css`
+    [part='scroller'] {
+      display: flex;
+      flex-direction: column;
+      overflow: auto;
+      -webkit-overflow-scrolling: touch;
```

Comment:
This is likely redundant, as this has been unnecessary since Safari 13: https://developer.apple.com/documentation/safari-release-notes/safari-13-release-notes

![Screen Shot 2021-11-25 at 12 17 16](https://user-images.githubusercontent.com/66382/143423073-7f39a028-8544-4d83-8564-9aad4122c09a.png)

Reply:
Does this also apply to `overflow: auto`? The release notes are a bit unclear 🤔

Reply:
Yeah, I think so. But I can test that quickly as well.

Reply:
Yes, the same applies for `overflow: auto`

Reply:
Great, thanks! I will update the other PR to remove this.

Reply:
Done in https://github.com/vaadin/web-components/pull/3106/commits/c3a420ca20e79d839fdc9ef744e0e2bc05ef20aa